pkg/ebpf/tracee.go

```diff
@@ -1534,64 +1573,76 @@ func (t *Tracee) triggerMemDump(event trace.Event) error {
 		return nil
 	}
 
-	printMemDumpFilters := t.config.Filter.ArgFilter.GetEventFilters(events.PrintMemDump)
-	if printMemDumpFilters == nil {
-		return fmt.Errorf("no address or symbols were provided to print_mem_dump event. " +
-			"please provide it via -t print_mem_dump.args.address=<hex address>" +
-			", -t print_mem_dump.args.symbol_name=<owner>:<symbol> or " +
-			"-t print_mem_dump.args.symbol_name=<symbol> if specifying a system owned symbol")
-	}
+	errArgFilter := make(map[int]error, 0)
 
-	eventHandle := t.triggerContexts.Store(event)
+	for filterScope := range t.config.FilterScopes.Map() {
```

Comment:
@AsafEitani we need to test the uprobe events with the new scope mechanisms (in particular this call I guess but all of them in general).